src/Api/Mastodon/Account.php

```diff
@@ -52,37 +53,52 @@ class Account
 	var $fields = null;
 	/** @var bool|null */
 	var $bot = null;
+	/** @var bool */
+	var $group;
+	/** @var bool */
+	var $discoverable;
+	/** @var string|null (Datetime) */
+	var $last_status_at = null;
 
 	/**
-	 * Creates an account record from a contact record. Expects all contact table fields to be set
+	 * Creates an account record from a public contact record. Expects all contact table fields to be set.
 	 *
-	 * @param array $contact   Full contact table record
-	 * @param array $apcontact Full apcontact table record
+	 * @param BaseURL $baseUrl
+	 * @param array   $publicContact Full contact table record with uid = 0
+	 * @param array   $apcontact     Optional full apcontact table record
 	 * @return Account
 	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
 	 */
-	public static function createFromContact(array $contact, array $apcontact = [])
+	public static function create(BaseURL $baseUrl, array $publicContact, array $apcontact = [])
```

Comment:
We really have to talk about the static creation of classes with dynamic created dependencies.. I believe this will lead to drawbacks on a long run - similar problems like with the mix of using dynamic classes in static functions. But I don't have a good way to explain my thoughts (yet). I'll create a forum post with hopefully good examples